Packs/Cyberint/Integrations/Cyberint/Cyberint.py

```diff
@@ -504,20 +540,53 @@ def fetch_incidents(client: Client, last_run: Dict[str, int],
                         attachments.append(tmp_attachment)
 
         alert["attachments"] = attachments
-        alert_csv_id = alert.get('alert_data', {}).get('csv', {}).get('id', '')
-        if alert_csv_id:
-            extracted_csv_data = extract_data_from_csv_stream(client, alert_id,  # type: ignore
-                                                              alert_csv_id)
-            alert['alert_data']['csv'] = extracted_csv_data
+        incident = {}
+
+        incident_csvs = dict_safe_get(alert, ['alert_data', 'csv', 'content']) or []
+
+        if not fetch_assign_multiple:

```

Comment:
```suggestion
        if  fetch_assign_multiple:

```
It seems that the logic here is to duplicate alert. In that case, isn't **fetch_assign_multiple** True?

Reply:
Now yes, before the name is changed this was the right logic.